src/storage/src/source/metrics.rs

```diff
@@ -93,6 +94,11 @@ impl SourceSpecificMetrics {
                 help: "A counter representing how many times we have failed to commit offsets for a source",
                 var_labels: ["source_id"],
             )),
+            inmemory_remap_bindings: registry.register(metric!(
+                name: "mz_source_inmemory_remap_bindings",
+                help: "The number of in-memory remap bindings that reclocking a time needs to iterate over.",
+                var_labels: ["source_id"],
```

Comment:
we should have this parameterized by `worker_id`, otherwise the various workers will just fight with each other

Reply:
Done